### PR TITLE
Remove the compose operator from the fn API table

### PR DESCRIPTION
**Pull request by JanuszL**
- as the compose operator is not supported in the functional
  API this PR removes it from the supported OP table

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:

**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- as the compose operator is not supported in the functional
  API this PR removes it from the supported OP table

### Affected modules and functionalities:
- operators table

### Key points relevant for the review:
- look and feel

### Tests
- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [x] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [x] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

File: docs/operations_table.py [hunk 1]
```diff
@@ -80,6 +85,8 @@ def fn_to_op_table(out_filename):
             if m is not None and hasattr(m, op_name):
                 op_string = link_formatter.format(op = op_full_name, module = module_name)
                 fn_string = link_formatter.format(op = to_fn_name(op_full_name), module = to_fn_module(module_name))
+        if op_name.lower() in removed_ops:
+            fn_string = "NA"
```
Comment (mzient) [inline docs/operations_table.py:89]: Why `.lower()`? I think it should use either ops style OperatorName or fn style operator_name, not yet another variant.

Comment (jantonguirao) [reply docs/operations_table.py:89]: I'd recommend creating a function `def has_fn_variant(op_name)` or similar
```
def has_fn_variant(op_name):
    no_fn_list = ['Compose']
    return False if op_name in no_fn_list else True   
```
then use it here and in line 106

[the remark this answers is not in the document]
Comment (JanuszL) [reply docs/operations_table.py]: Fixed

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
The most recent:
image

**Review by mzient — APPROVED** [review]

**Review by jantonguirao — APPROVED** [review]

**Pull request merged**